st2client/st2client/models/core.py

```diff
@@ -240,7 +240,8 @@ def query(self, **kwargs):
         if 'limit' in kwargs and kwargs.get('limit') <= 0:
             kwargs.pop('limit')
         token = kwargs.get('token', None)
-        params = {}
+        params = kwargs.get('params', {})
```

Comment:
Sigh, yeah, wish we could just port everything to explicitly use `params` argument instead of abusing kwargs...